lessons/Data-Science/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,278 @@
+
+}
+
+@QandA{
+@Q{After looking at the decision trees of your classmates: What do these trees all have in common? How are they different?}
+@A{Answers will vary. Each tree will have twice as many branches as nodes. Many trees will have the same number of nodes, although probably not all. Many questions asked will likely be the same, but not every question.}
+@Q{Are all of the trees equally efficient?}
+@A{The trees are probably similarly efficient, requiring either five or six nodes, including the root. Five nodes is more efficient than six.}
+}
+
+You might be wondering: Would it be possible to make this tree any more efficient? If so, why not?!
+
+@lesson-instruction{
+- With a partner, complete @printable-exercise{comparing-trees.adoc}.
+}
+
+As we increase a tree's efficiency, we can lose accuracy. Conversely, if we focus *too* much on accuracy - making our decision trees bigger and more complex - we risk __overfitting__. Overfitting happens when we teach a model the specific quirks of one particular dataset, preventing it from making reliable predictions about new data.
+
+Decision trees can accurately label and categorize the inputs that they are trained to label and categorize! This tree—which was 100% accurate on the trained dataset—falters when we offer it inputs that are either *unknown*, like the chopstick, or *ambiguous*, like the spork. In both instances, the tree offers up its best guess at a label. __The only way this tree stands a chance of correctly identifying a chopstick or a spork is if we offer it more training!__
+
+As we built our decision trees, we were able to draw on everything we know about every knife, spoon, spork, plate, bowl or mug that we have ever seen. If you were asked to create a decision tree to identify common animals or foods, you could probably do that without much difficulty as well.
+
+Computers, however, build decision trees using only the data we provide... and that data can sometimes be messy. As a result, we may end up with models that are not 100% accurate.
+
+Imagine that you are tasked with building a decision tree that can determine an iris' species (_setosa_, _virginica_, and _versicolor_) based on the varying plant measurements. It would be far more challenging to build this tree than the one we just made that classifies common tableware!
+
+@centered-image{images/iris-data.png}
+
+=== Synthesize
+
+@QandA{
+
+@Q{How is playing 20 questions to figure out a mystery item _similar_ to the predicting and recommending that a decision tree enables?}
+@A{Decision trees, like 20 Questions participants, need to avoid questions that don't provide information. Decision trees, like 20 Questions participants, sometimes fail to identify an object correctly. Familiarity with the dataset improves the accuracy of both a decision tree and a 20 Questions participant.}
```

Comment:
"questions that don't provide information"? 

I think I know what that means based on the lesson, but it would be a lot more helpful if somewhere earlier in the lesson there was a sentence that made it clear that **the answer to each question adds information that helps us categorize an object**